documentation/src/main/doc/modules/kafka/pages/protobuf-configuration.adoc

```diff
@@ -0,0 +1,45 @@
+[#kafka-protobuf-configuration]
+== Using Google Protobuf serializer/deserializer
+
+If you are using https://developers.google.com/protocol-buffers/[Protocol Buffers] serializer/deserializer, please note the following configuration properties.
+
+=== For https://docs.confluent.io/platform/current/schema-registry/serdes-develop/serdes-protobuf.html#serdes-and-formatter-protobuf[Confluent] Schema Registry
+
+Confluent protobuf library is `io.confluent:kafka-protobuf-serializer`.
+Note that this library is not available in Maven Central, you need to use the https://docs.confluent.io/clients-kafka-java/current/overview.html[Confluent Maven repository].
+
+==== Consumer
+
+[options="header"]
+|===
+| Property | Recommended value
+| value.deserializer | io.confluent.kafka.serializers.protobuf.KafkaProtobufDeserializer
```

Comment:
```suggestion
| value.deserializer | `io.confluent.kafka.serializers.protobuf.KafkaProtobufDeserializer`
```

Reply:
The Avro guide doesn't have backticks here either. I considered adding them, but figured it's probably better to avoid them, to allow for word wrapping. But I don't really mind.

Reply:
From another issue we had recently, it does not seem to avoid word-wrapping...

Reply:
Ah interesting! In any case, we don't have it for Avro, so I'd be fine with leaving this as is :-)